wp-web-push/lib/style/subscription_button.template.css

```diff
@@ -122,6 +122,10 @@
   pointer-events: none;
 }
 
+#webpush-subscription img {
```

Comment:
This is very general for a very specific case. My recommendation is that you name (with an id or a class) the image and use a more specific selector to fix it or, you can adjust margin for p elements inside the dialog (last rule in the CSS file) adding some air below each one and a special rule for the last one removing this space. This is a more generic solution but I would bet for the first one as nothing more is needed right now.